qaframework-bdd-tests/src/main/java/org/openmrs/contrib/qaframework/page/VisitNotePage.java

```diff
@@ -87,6 +89,15 @@ public void enterSecondaryDiagnosis(String diag) {
 		clickOn(UI_MENU_ITEM);
 	}
 
+	public void removeDiagnosis() {
+		clickOn(REMOVE_DIAGNOSIS_BUTTON);
+	}
+
+	public void clearNote() {
```

Comment:
@Muta-Jonathan the method name does not reflect what is being performed.

Reply:
@kdaud any Suggestion for the name .....of the method

Reply:
Have you taken a look at what the method is performing?

Reply:
edited @kdaud